spec/requests/requests_spec.rb

```diff
@@ -258,7 +258,7 @@
 
       FactoryBot.create(:classification_department_with_tags)
 
-      t = Classification.where(:description => 'Department', :parent_id => 0).includes(:tag).first
+      t = Classification.where(:description => 'Department', :parent_id => nil).includes(:tag).first
```

Comment:
This looks a bit strange because the category should have `:parent_id => 0` in Classifications. Is it possible that the factory can be fixed to ensure that the category is created that way instead of this change?